command/main.go

```diff
@@ -24,6 +24,17 @@ type VaultUI struct {
 	detailed bool
 }
 
+const (
+	envFlagOutputCurlString = "output-curl-string"
+	envFlagOutputPolicy     = "output-policy"
+	envFlagFormat           = "format"
+	envFlagDetailed         = "detailed"
+)
+
+var envFlags = []string{
+	envFlagOutputCurlString, envFlagOutputPolicy, envFlagFormat, envFlagDetailed,
```

Comment:
As above, `globalFlagXXX` would be slightly clearer to me, but feel free to disagree